submissions/VBystrov/dom/js/app.js

```diff
@@ -0,0 +1,50 @@
+import creatures from './tower.js';
+
+document.addEventListener('DOMContentLoaded', () => {
+  const navigation = document.getElementById('navigation');
+  const main = document.getElementById('main');
+  const buttonText = document.getElementById('button-text');
+
+  navigation.addEventListener('click', (e) => {
+    if (e.target.classList.contains('nav-item')) {
+      buttonText.textContent = e.target.textContent;
+
+      // main.innerHTML = '';
+      let desc = main.getElementsByClassName('description');
+      if (desc.length) {
+        desc[0].remove();
+      }
+      let newDesc = createDescription(e.target.textContent, creatures);
+      main.appendChild(newDesc);
+      // main.insertAdjacentElement('beforeend', newDesc);
+
+      let active = navigation.getElementsByClassName('active');
+      if (active.length) {
+        active[0].classList.remove('active');
+      }
+      e.target.classList.add('active');
+    }
+  });
+
+  function createDescription(creatureName, creatures) {
```

Comment:
I'd recommend to return string here (not an element). Please use template [strings](https://developer.mozilla.org/uk/docs/Web/JavaScript/Reference/Template_literals) for it.
It'll help you to get rid of redundant DOM operations.

Reply:
Done.